sros2/test/sros2/commands/security/verbs/test_list_keys.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2019 Canonical Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import tempfile
+
+from ros2cli import cli
+
+
+def test_list_keys(capsys):
+    with tempfile.TemporaryDirectory() as keystore_dir:
+        with capsys.disabled():
+            # First, create the keystore
+            assert cli.main(argv=['security', 'create_keystore', keystore_dir]) == 0
```

Comment:
Considering this is not a test for the `create_keystore` verb, maybe we just use the API  call directly (`from sros2.api import create_keystore`)? I don't have a strong preference, but it might read a little better.

Same for other call to create keystore below.

Reply:
Agreed, I like that better as well-- done.